packages/events/src/events.ts

```diff
@@ -0,0 +1,264 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/listener
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// tslint:disable-next-line:no-any
+export type Event = any;
+
+/**
+ * Async event listener function
+ */
+export type ListenerFunction = (
+  /**
+   * Event type
+   */
+  eventType: string,
```

Comment:
Personally, I'd prefer to use `eventName` instead of `eventType`. In my mind, a "type" usually refer to concepts like Number, String, Product (i.e. a constructor function). OTOH, "name" is almost always a string.

Reply:
Added `EventName<T>`